File: Database/Assembly_SRPG_JP/PlayerParam.cs
// Decompiled with JetBrains decompiler
// Type: SRPG.PlayerParam
// Assembly: Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
// MVID: 85BFDF7F-5712-4D45-9CD6-3465C703DFDF
// Assembly location: S:\Desktop\Assembly-CSharp.dll

namespace SRPG
{
  public class PlayerParam
  {
    public OInt pt;
    public OInt ucap;
    public OInt icap;
    public OInt ecap;
    public OInt fcap;

    public bool Deserialize(JSON_PlayerParam json)
    {
      if (json == null)
        return false;
      this.pt = (OInt) json.pt;
      this.ucap = (OInt) json.ucap;
      this.icap = (OInt) json.icap;
      this.ecap = (OInt) json.ecap;
      this.fcap = (OInt) json.fcap;
      return true;
    }
  }
}

File: Database_V2/ParamFunctions/PlayerParam.py
def PlayerParam(json):
    this={}#PlayerParamjson)
    #if(json==null)
    #returnfalse
    if 'pt' in json:
        this['pt'] = json['pt']
    if 'ucap' in json:
        this['ucap'] = json['ucap']
    if 'icap' in json:
        this['icap'] = json['icap']
    if 'ecap' in json:
        this['ecap'] = json['ecap']
    if 'fcap' in json:
        this['fcap'] = json['fcap']
    #returntrue
    return this
